src/store/messagesStore.js

```diff
@@ -1351,9 +1355,9 @@ const actions = {
 			})
 			// The response return an array with the reaction details for this message
 			const response = await addReactionToMessage(token, messageId, selectedEmoji)
-			// We replace the reaction details in the reactions store and wipe the old
-			// values
-			context.dispatch('updateReactions', {
+
+			const reactionsStore = useReactionsStore()
```

Comment:
I'd move this and next action fully to reactionsStore. Apart from updating the message object, it doesn't really belong to messagesStore. You can use this workaround for now, until messagesStore isn't migrated to Pinia:
```js
const store = useStore()
store.commit('addReactionToMessage', payload)
```